# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/iproute2/min.patch.musl
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

diff --git a/ip/ipstats.c b/ip/ipstats.c
index 5cdd15a..7ccfae8 100644
--- a/ip/ipstats.c
+++ b/ip/ipstats.c
@@ -6,6 +6,8 @@
 #include "utils.h"
 #include "ip_common.h"
 
+#define MIN(a,b) (((a)<(b))?(a):(b))
+
 struct ipstats_stat_dump_filters {
 	/* mask[0] filters outer attributes. Then individual nests have their
 	 * filtering mask at the index of the nested attribute.
